tests/models.py

```diff
@@ -10,21 +10,21 @@
 D2 = {1: 2, 2: 4, 3: 6, 4: 8, 5: 10}
 
 
-class TestCopyDataType(str):
+class FakeCopyDataType(str):
```

Comment:
What's the purpose of these name changes? They seem to be adding a lot of noise for no reason.